lib/Jsrt/ChakraCommon.h

```diff
@@ -2434,6 +2496,22 @@ typedef unsigned short uint16_t;
             _In_opt_ JsPromiseContinuationCallback promiseContinuationCallback,
             _In_opt_ void *callbackState);
 
+    /// <summary>
+    ///      Returns a value that indicates whether an object is a constructor.
+    /// </summary>
+    /// <remarks>
+    ///     Requires an active script context.
+    /// </remarks>
+    /// <param name="object">The object to test.</param>
+    /// <param name="isConstructor">If the object is a constructor, <c>true</c>, <c>false</c> otherwise.</param>
+    /// <returns>
+    ///     The code <c>JsNoError</c> if the operation succeeded, a failure code otherwise.
+    /// </returns>
+    CHAKRA_API
```

Comment:
Can we get this and the other typedefs in ChakraCore.h? ChakraCommon is for shared APIs between Chakra and ChakraCore.